controllers/infra/validatingwebhookconfiguration/validatingwebhookconfiguration_controller.go

```diff
@@ -101,14 +105,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Re
 	ctx = pkgcfg.JoinContext(ctx, r.Context)
 
 	if req.Name != spqutil.ValidatingWebhookConfigName {
-		r.Logger.Error(nil, "Reconcile called for unexpected object", "name", req.Name)
+		logr.FromContextOrDiscard(ctx).Error(nil, "Reconcile called for unexpected object")
```

Comment:
Same question as before, should we ever be using `logr.FromContextOrDiscard` anymore?